README.md

```diff
@@ -37,22 +37,35 @@ Face capture uses the webcam by default for capturing live photos of users. File
 
 In order to start integration, you will need the **API token**. You can use our [sandbox](https://documentation.onfido.com/#testing) environment to test your integration, and you will find the sandbox token inside your [Onfido Dashboard](https://onfido.com/dashboard/api/tokens).
 
-### 2. Generating a JSON Web Token
+### 2. Creating an applicant
 
-For security reasons, instead of using the API token directly in you client-side code, you will need to generate and include a short-lived JSON Web Token (JWT)\* everytime you initialise the SDK. The easiest way to generate a JWT is via the [JWT endpoint](https://documentation.onfido.com/#json-web-tokens) in the Onfido API:
+With your API token, you should create an applicant by making a request to the [create applicant endpoint](https://documentation.onfido.com/#create-applicant) from your server:
 
 ```shell
-$ curl https://api.onfido.com/v2/jwt?referrer=YOUR_WEBPAGE_URL \
+$ curl https://api.onfido.com/v2/applicants \
+    -H 'Authorization: Token token=YOUR_API_TOKEN' \
+    -d 'first_name=John' \
+    -d 'last_name=Smith'
+```
+
+You will receive a response containing the applicant id which will be used to create a JSON Web Token.
+
+### 3. Generating an SDK token
+
+For security reasons, instead of using the API token directly in you client-side code, you will need to generate and include a short-lived JSON Web Token ([JWT](https://jwt.io/)) every time you initialise the SDK. The easiest way to generate an SDK Token is via the [SDK Token endpoint](https://documentation.onfido.com/#sdk-tokens) in the Onfido API:
+
+```shell
+$ curl https://api.onfido.com/v2/sdk_token?referrer=YOUR_WEBPAGE_URL&applicant_id=YOUR_APPLICANT_ID \
```

Comment:
I know, as I said I will use the API style instead of the query string. However it does work even if I use query string instead of form data. Isn't query string just another way of specifying the body parameters? Or is there something I am missing?
The syntax would be
```shell
curl -X POST 'https://api.onfido.com/v2/sdk_token?referrer=<referrer_pattern>&applicant_id=<applicant_id>' -H 'Authorization: Token token=<your_api_token>'
```
Again, I will use the form data as in the API documentation, I'm just trying to understand what's wrong with using query string.